### DISCUSSION TITLE
Is aptabase self-host totally independient?

### DISCUSSION THREAD

**Discussion by JOSEW383** [aptabase/aptabase#106] [Q&A] [answered]
When I host the aptabase server and add an app, it generates a key that I have to add to my app. How does the app know where the database or the address of my server is? Does that mean that there is a connection with the aptabase servers and that therefore, if it goes down, my metrics will also go down?

**Comment by cristipufu** [comment]
You have to configure the server url in the initialization options of the SDK

**Reply by JOSEW383** [reply]
Where? I have not seen any documentation on where to indicate the host.
[https://github.com/aptabase/tauri-plugin-aptabase
](https://github.com/aptabase/tauri-plugin-aptabase)
I guess it would be along these lines although I have not seen any documentation on this.
`.plugin(tauri_plugin_aptabase::Builder::new("<YOUR_APP_KEY>").build()) // 👈 this is where you enter your App Key`

**Reply by cristipufu** [reply, marked as answer]
https://github.com/aptabase/tauri-plugin-aptabase/blob/main/src/config.rs#L28 - InitOptions.host
